configure

```diff
@@ -115,6 +115,10 @@ if [[ ! -e "$SWIG_PATH" ]]; then
   echo "Can't find swig.  Ensure swig is in \$PATH or set \$SWIG_PATH."
   exit 1
 fi
+# Convert swig path to Windows style before writing into swig_path
+if is_windows; then
+  SWIG_PATH="$(cygpath -m "$SWIG_PATH")"
```

Comment:
The problem is actually when writing PYTHON_BIN_PATH into `tools/bazel.rc`,  if we use 
`-w`, the bazel.rc file will look like:
```
build --define PYTHON_BIN_PATH="C:Program FilesAnaconda3python"
test --define PYTHON_BIN_PATH="C:Program FilesAnaconda3python"
```
So I feel using `-m` is a safer choice.